src/main/java/teammates/common/datatransfer/FeedbackSessionQuestionsBundle.java

```diff
@@ -129,4 +156,25 @@ public void resetAllResponses() {
             questionResponseBundle.put(question, new ArrayList<FeedbackResponseAttributes>());
         }
     }
+
+    public String getNameForEmail(String email) {
+        String name = emailNameTable.get(email);
+        if (name == null || name.equals(Const.USER_IS_MISSING)) {
+            return Const.USER_UNKNOWN_TEXT;
+        } else if (name.equals(Const.USER_IS_NOBODY)) {
+            return Const.USER_NOBODY_TEXT;
+        } else if (name.equals(Const.USER_IS_TEAM)) {
+            return getTeamNameForEmail(email);
+        } else {
+            return name;
+        }
+    }
+
+    public String getTeamNameForEmail(String email) {
+        String teamName = emailTeamNameTable.get(email);
+        if (teamName == null || email.equals(Const.GENERAL_QUESTION)) {
```

Comment:
what is `GENERAL_QUESTION`? Is it not in `Const` yet? Can help me understand how an email is equal to a question?